styles/utility.css

```diff
@@ -700,4 +821,85 @@ textarea {
   hr {
     margin-top: 70px;
   }
+
+  .modal-container {
+    display: flex;
+    justify-content: space-evenly;
+    left: 0;
+    top: 0;
+  }
+
+  .modal-wrapper {
+    background: #fff;
+    border-radius: 8px;
+    height: 100%;
+    width: 920px;
+    padding: 20px 20px 20px;
+    margin: 50px 20px 20px 20px;
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+  }
+
+  .project-image {
+    margin-top: 20px;
+    content: url('../images/Snapshoot\ Portfolio\ Desktop.png');
+  }
+
+  .modal-project-desc-tech-details {
+    margin-top: 15px;
+    text-align: left;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .project-title {
+    font-size: 32px;
+    font-weight: 600;
+    color: #344563;
+    padding-left: 20px;
+    line-height: 32px;
+  }
+
+  .tech-list-modal-container {
+    display: flex;
+    width: 100%;
+    margin: 12px 20px 0 0;
+    justify-items: normal;
+    flex-flow: row nowrap;
+    align-items: center;
+  }
+
+  li.project-technologies {
+    border-radius: 8px;
+    background: #ebf0ee;
+    padding: 12px;
+  }
+
+  li.project-technologies p {
+    color: #3a4a42;
+    font-weight: 600;
+    font-size: 12px;
+  }
+
+  p.modal-text-content {
+    margin: 20px 12px 12px;
+    line-height: 20px;
+  }
+
+  .modal-buttons .btn {
+    display: inline-flex;
+    flex-direction: row;
+    width: 45%;
+    justify-content: space-between;
+    margin-top: 20px;
+    background-color: #36b37e;
+    color: #fff;
+    font-size: 16px;
+    letter-spacing: 0.03em;
+    text-align: center;
+    border-radius: 4px;
+    align-self: center;
+  }
```

Comment:
You did a great job in mobile design, only just little adjustments and will look amazing, please note that the buttons in your design are not inside the white container as in Figma design and that your close button is in a position that doesn't fit with the Figma design.

Your Design: 
![image](https://user-images.githubusercontent.com/82678907/152262159-d7f6a751-fd3e-40d4-b8c9-884580ca2f65.png)

![image](https://user-images.githubusercontent.com/82678907/152262277-a75d1097-d972-439c-8fbd-0cc21e538ed3.png)


Figma Design:
![image](https://user-images.githubusercontent.com/82678907/152262017-4b4ba699-7684-4778-b30a-063cafc4ee94.png)

- [ ] Kindly fix your close button and source-live buttons, doing this you will attract more hiring managers for sure. 😄

Reply:
- [ ] Sadly, close button position is still wrong, please try to put it inside the modal window: 
![image](https://user-images.githubusercontent.com/77038610/152541640-838df998-e86c-44eb-8be8-2fc049ae9bf3.png)

Reply:
The close button for both desktop and mobile is now in the right direction.

Thanks for the review 😄